cogs/tags.py

```diff
@@ -157,6 +160,41 @@ def get_tag(self, server, name, *, redirect=True):
                 raise RuntimeError('Tag not found.')
             raise RuntimeError('Tag not found. Did you mean...\n' + '\n'.join(possible_matches))
 
+    async def user_overlimit(self, ctx):
+        """Check to see if user has too many tags.
+
+        Returns:
+        --------
+        bool
+            True if too many tags, False if okay.
+        """
+        owner = ctx.message.author
+        server = ctx.message.server
+
+        if owner.id == ctx.bot.settings.owner:
+            # No limit for bot owner
+            return False
+
+        if server:
+            # No limit for mods and admins of server.
+            mod_role = ctx.bot.settings.get_server_mod(server).lower()
+            admin_role = ctx.bot.settings.get_server_admin(server).lower()
+            roles = [role.name for role in owner.roles]
+            if admin_role in roles or mod_role in roles:
```

Comment:
Maybe some of these role checks should be configurable like with `MOD_TAG_LIMIT = False etc`?

Reply:
Sure, so something similar to what we have for word filter, where we can toggle filtering on mods and above?

Reply:
yeas

Reply:
Hmm. I don't know how much of that would be worth it, since mods and up has tags that other users have requested, so unless we somehow transfer those to the user when they get to sensei or overhaul the tags to allow mods to change ownership or something...

Reply:
We can look into that in a future PR, and see how feasible it would be first? If we ignore this for now, is there anything else you guys think I should look at?